GettingStarted.md

```diff
@@ -95,12 +95,11 @@ $ vagrant status
 Current machine states:
 
 default                   running (libvirt)
+```
 
 The Libvirt domain is running. To stop this machine, you can run
 `vagrant halt`. To destroy the machine, you can run `vagrant destroy`.
 
-```
```

Comment:
ohh.. sorry.. didn't realized you switched it to vagrant command.. i had a virsh command there originally